.github/workflows/continuous-delivery.yml

```
@@ -14,33 +14,25 @@ jobs:
         uses: actions/checkout@v2
         with:
           persist-credentials: false
-      - name: Build
-        run: |
-          docker build \
-            --tag docker.pkg.github.com/ridedott/flatbuffers-builder-docker/flatbuffers-builder:latest \
-            --tag registry.hub.docker.com/ridedott/flatbuffers-builder:latest \
-            .
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
       - env:
+          DOCKER_BUILDKIT: 1
+          DOCKER_HUB_REGISTRY_USERNAME: ${{ secrets.DOCKER_HUB_REGISTRY_USERNAME }}
+          DOCKER_HUB_REGISTRY_PASSWORD: ${{ secrets.DOCKER_HUB_REGISTRY_PASSWORD }}
           GITHUB_REGISTRY_USERNAME: ${{ github.actor }}
-          GITHUB_REGISTRY_PASSWORD: ${{ secrets.GITHUB_TOKEN_DOTTBOTT }}
-          DOCKER_HUB_REGISTRY_USERNAME:
-            ${{ secrets.DOCKER_HUB_REGISTRY_USERNAME }}
-          DOCKER_HUB_REGISTRY_PASSWORD:
-            ${{ secrets.DOCKER_HUB_REGISTRY_PASSWORD }}
-        name: Release
+          GITHUB_REGISTRY_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
```

This one seems to be new? Is it set for the repo?

---

ah it's set on org level, nevermind

---

or actually GITHUB_TOKEN is not set, probably needs to be:
```suggestion
          GITHUB_REGISTRY_PASSWORD: ${{ secrets.GITHUB_TOKEN_DOTTBOTT }}
```
Or maybe `DOTTBOTT_TOKEN` 🤔

---

`GITHUB_TOKEN` Seems to not be here though?